libs/parallelism/algorithms/include/hpx/parallel/algorithms/is_sorted.hpp

```diff
@@ -447,7 +447,7 @@ namespace hpx { namespace parallel { inline namespace v1 {
 
```

Comment:
Would you mind adding the `tag_fallback_invoke` include to this file. It was already missing the `tag_invoke` include.

Reply:
done